modules/formulize/include/functions.php

```diff
@@ -4995,12 +5004,17 @@ function _buildConditionsFilterSQL($filterId, &$filterOps, &$filterTerms, $filte
       } else {
 	$filterOps[$filterId] = 'IS';
       }
-    } 
```

Comment:
Again, I'm not sure what the replacement code in this case is meant to do...this is a part of the code that has been perfected a fair bit over recent years, and I don't think we were doing anything with it in the functionality the UCOSP branch was developing. Since this doesn't relate to the template work, I would pull this and the newFilterOps changes above, out of the branch.

Reply:
same as above